server/authentication/auth.js

```diff
@@ -15,9 +16,14 @@ function authenticationMiddleware(signInService) {
                 res.locals.token = token;
                 return next();
             }
-            const systemToken = await signInService.getClientCredentialsTokens(username);
-            res.locals.token = systemToken.token;
-            return next();
+            try {
+                const systemToken = await signInService.getClientCredentialsTokens(username);
+                res.locals.token = systemToken.token;
+                return next();
+            } catch (error) {
+                logger.error('Unable to get client credentials token for RO', error.stack);
+                return res.redirect('/logout');
```

Comment:
is the right place to redirect them?  Won't they logout, login again and then be logged out automatically?

Reply:
Yes. Not an ideal solution. Shouldn't really occur in the wild. Maybe this full PR is unnecessary, will close